config.yaml

```diff
@@ -286,17 +286,18 @@ options:
     description: |
       Verify that a specific cis audit profile was used for the audit. If not specified the
       profile will be extracted from '/var/log/cloud-init-output.log', fallback is 'level1_server'.
-      Options are: '' (disable profile check), 'level1_server', 'level2_server',
-                   'level1_workstation' or 'level2_workstation'
-      This option cannot be used with cis_audit_tailoringfile at the same time.
+      To see the valid options to this configuration, check the documentation at
+      https://ubuntu.com/security/certifications/docs/usg/cis/compliance
```

Comment:
Please retain the options here.  You can still link to the upstream docs, but it's nice UX to have the available profile options handy.  Also, the profile options accepted here are different to the profile names in the upstream docs.

Reply:
The options can change depending on the Ubuntu series. For focal and jammy the options has a `cis_` prefix E.g: `cis_level1_workstation`. On the other hand, for bionic is without the prefix. E.g: `lvl1_workstation`

That is why I don't want to have the options because they can change and the options that had before it's not always true. However I think we can add this info in description

Reply:
on https://github.com/canonical/charm-nrpe/pull/190 I've requested that we keep backward compatibility so the options will not change.  Either way, changing docs for the profile options is not really in scope for this PR.